docs/userGuide/usingPlugins.md

```diff
@@ -115,4 +115,13 @@ MarkBind has a set of built-in plugins that can be used immediately without inst
 
 <include src="plugins/filterTags.mbdf" />
 
+### Advanced: Default plugins
+
+Markbind has a set of default plugins that it uses to carry out some of its features. These are enabled by default for every project and should be left alone.
+
+Default Plugin | Functionality
+--- | ---
+`anchors` | Attaches anchor links to the side of headings.
+
+Although not advised, you can disable these by passing `"off": true` in the `pluginsContext`.

```

Comment:
Give a brief code snippet example, so that the user can have a better idea on how this is done (btw the code snippet given in your OP doesn't work because of missing/mismatch quotation marks).